src/utils/DataProcessingUtils.js

```diff
@@ -27,22 +37,24 @@ const VALUE_MAPPERS :'VALUE_MAPPERS' = 'VALUE_MAPPERS';
 const { FullyQualifiedName } = Models;
 
 function getPageSectionKey(page :number, section :number) :string {
-
   return `page${page}_section${section}`;
 }
 
 declare type UUID = string;
+type IndexOrId = number | UUID;
+type EntityData = { [UUID] :any[] };
+type EdgeDefinition = [string, IndexOrId, string, IndexOrId, string, EntityData];
 
-export type EntityAddress = {|
+type EntityAddress = {|
   entityIndex ? :number;
   entityKeyId ? :UUID;
   entitySetName :string;
   propertyTypeFQN :FQN;
 |};
 
 function getEntityAddressKey(
-  indexOrEntityKeyId :number | UUID,
-  entitySetName :string,
+  indexOrEntityKeyId :IndexOrId,
+  entitySetName :FQN | string,
```

Comment:
I don't follow... in what case would `entitySetName` ever be an `FQN`?

Reply:
I did this to support the usage of app types that follow the fqn pattern.